docker/worker/oss_fuzz.py

```diff
@@ -223,6 +222,14 @@ def set_bug_attributes(bug, regress_result, fix_result):
   bug.regressed = regress_result.commit or ''
   bug.fixed = fix_result.commit or ''
 
+  bug.affected_ranges = [
+      osv.AffectedRange(
+          type='GIT',
```

Comment:
can we define a enum for 'GIT' and other type.

Reply:
enums are hard to deal with for strings (have to use Type.GIT.value) and we'll be taking them as a strings from the JSON anyway. 

I added a validator here to the Datastore type so that these are always valid.